packages/input-amount/test/lion-input-amount.test.js

```diff
@@ -119,6 +119,29 @@ describe('<lion-input-amount>', () => {
     ).to.equal('TL');
   });
 
+  it('sets serializedValue in correct currency decimals', async () => {
```

Comment:
Maybe formulate it as 'the serializedValue should reflect what the end user sees on his screen (so serializedValue should be aligned with viewValue)'. 

Also, the goal of serializedValue is to be a serialized version of the modelValue. That means serializedValue <=> modelValue...

If we want to achieve that as well, it might be better to cut the number of decimals already in the parsing phase (serialize function could probably stay as was before, then).

Sorry for the confusion, I thought serializer function above was going to solve it, forgetting modelValue would be out of sync now.... :)

Reply:
No problem, good to keep ourselves sharp

Reply:
So that would probably mean adding `decimals`to options here: https://github.com/ing-bank/lion/blob/master/packages/input-amount/src/parsers.js#L124

As a final step in the method, we can cut the number of decimals when provided as option.
To prevent rounding errors, we could probably use Intl (like we do in the formatter) to get the right decimals.